.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,6 +8,6 @@
 - fixes #123
 
 ------
-- [ ] Updated [changelog](CHANGELOG.md#unreleased)
-- [ ] Updated [migration guide](MIGRATIONS.md#unreleased) if applicable
+- [ ] Updated [changelog](https://github.com/scalableminds/webknossos/blob/master/CHANGELOG.md#unreleased)
```

Comment:
Can we go relative? `../blob/master/CHANGELOG.md`

Reply:
yes, tested it :+1: